<launch>
  <node pkg="kinect_face_detection" type="face_detector" name="face_detector" output="screen">
  	 <param name="classifier_name" type="string" value="frontalface" />
	 <param name="classifier_filename" type="string" value="$(find opencv2)/opencv/share/opencv/haarcascades/haarcascade_frontalface_alt.xml" />
	 <param name="classifier_reliability" type="double" value="0.9"/>
  	 <param name="do_continuous" type="bool" value="true" />
  	 <param name="do_publish_faces_of_unknown_size" type="bool" value="true" />	
	 <param name="do_display" type="string" value="local" />
  	 <param name="output" type="string" value="none" />
  </node>

</launch>
